assets/js/blocks/reviews/reviews-by-category/block.tsx

```diff
@@ -10,20 +10,20 @@ import {
 	ToggleControl,
 	withSpokenMessages,
 } from '@wordpress/components';
-import PropTypes from 'prop-types';
 import ProductCategoryControl from '@woocommerce/editor-components/product-category-control';
 import { Icon, commentContent } from '@wordpress/icons';
 
 /**
  * Internal dependencies
  */
 import EditorContainerBlock from '../editor-container-block.js';
-import NoReviewsPlaceholder from './no-reviews-placeholder.js';
+import NoReviewsPlaceholder from './no-reviews-placeholder';
 import {
 	getBlockControls,
 	getSharedReviewContentControls,
 	getSharedReviewListControls,
 } from '../edit-utils.js';
+import { ReviewsByCategoryEditorProps } from './types';
```

Comment:
```suggestion
import type { ReviewsByCategoryEditorProps } from './types';
```

Given that `ReviewsByCategoryEditorProps` is a type, using `import type ...` can lead to a slightly faster compile time.